server/index.js

```diff
@@ -14,6 +15,14 @@ app.use(bodyParser.json());
 app.use(cors());
 app.use('/rooms/:id', express.static(__dirname + '/../client/dist'));
 
+
+
+app.get('/', async(req, res) => {
+  couch.insertIntoCouchDB({"stayId": 14, "numBed": 5}, (status) => {
+    res.sendStatus(status);
+  });
+})
```

Comment:
Is there a reason this is the root route? Or is this for testing purposes

Reply:
Ya this was for testing purposes. I should take it out.